packages/blockchain-wallet-v4/src/exchange/utils.ts

```diff
@@ -39,17 +39,30 @@ export const coinToString = ({
     coinfig ? coinfig.displaySymbol : unit.symbol
   }`
 }
-
+// showNarrowSymbol - is used to show $ instead of US$
 export const fiatToString = ({
   digits = 2,
+  showNarrowSymbol = false,
   unit,
   value
 }: {
   digits?: number
+  showNarrowSymbol?: boolean
   unit: FiatType
   value: string | number
 }): string => {
-  const options = { currency: unit, minimumFractionDigits: digits, style: 'currency' }
+  const options = showNarrowSymbol
```

Comment:
Instead of having a big if/else with repeated info here you could do something like:
```
const options = {
  currency: unit,
  minimumFractionDigits: digits,
  style: 'currency',
  currencyDisplay: showNarrowSymbol ? 'narrowSymbol' :  undefined
}

```